node-sdk-sample/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "sdk-sample",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "commonjs",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint src",
+    "build": "tsc"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "ts-node": "^10.9.2"
```

Comment:
`ts-node` should be a dev dep since it's not required at runtime.

Or perhaps it can be removed altogether in favor of `tsx`?